workspaces/ballerina/ballerina-extension/package.json

```diff
@@ -1088,7 +1088,7 @@
         "copyFonts": "copyfiles -f ./node_modules/@wso2/font-wso2-vscode/dist/* ./resources/font-wso2-vscode/dist/",
         "copyVSIX": "copyfiles *.vsix ./vsix",
         "copyVSIXToRoot": "copyfiles -f ./vsix/*.vsix ../../..",
-        "download-ls": "node scripts/download-ls.js --prerelease",
+        "download-ls": "node scripts/download-ls.js",
```

Comment:
Do we have to do this change every time we have to switch between release and pre-release?